hr_timesheet_default_analytic_account/models/__init__.py

```diff
@@ -1,3 +1 @@
-from . import account_analytic_account
-from . import hr_employee
-from . import hr_timesheet_sheet
+from . import hr_employee, hr_timesheet_sheet, project_project
```

Comment:
We usually write one import per line in the community.

Reply:
ok, i didn’t know that. is this limited to `__init__.py` files? because i often see `from odoo import fields, model,…`, like in the example [here](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#32imports).

Reply:
done!